### DISCUSSION TITLE
version number in release 1.6.4

### DISCUSSION THREAD

**Discussion by kcning** [evilsocket/opensnitch#1065] [General]
Hello, thanks for the new release. I notice that however the version number in ```daemon/core/version.go``` in the tar ball remains 1.6.2. It's a quick fix but I thought I should drop you a note ;)

**Comment by gustavo-iniguez-goya** [comment]
Thank you @kcning !

Yes, that's expected. The daemon and the GUI have different versions. Sometimes are equal, but if only one of the components changed (like in this case), then I only increase that version (gui or daemon).

**Reply by kcning** [reply]
ok, thanks for the explanation!
